common/src/main/java/io/homeassistant/companion/android/common/sensors/DNDSensorManager.kt

```diff
@@ -72,12 +72,16 @@ class DNDSensorManager : SensorManager {
             NotificationManager.INTERRUPTION_FILTER_UNKNOWN -> STATE_UNKNOWN
             else -> STATE_UNKNOWN
         }
+        val icon = when (state) {
+            "off" -> "mdi:minus-circle-off"
+            else -> "mdi:minus-circle"
+        }
 
         onSensorUpdated(
             context,
             dndSensor,
             state,
-            dndSensor.statelessIcon,
+            icon,
```

Comment:
You can simplify this and remove the need for the `icon` variable defined above

```suggestion
            if (state == "off") dndSensor.statelessIcon else "mdi:minus-circle",
```